python/mp/color.py

```diff
@@ -6,6 +6,7 @@ def __init__(self, r=0, g=0, b=0, a=0, name='color'):
         self.g = g  # green
         self.b = b  # blue
         self.a = a  # alpha channel (not yet used)
+        self.brightness = 0xff;  # brightness used by APA102 LED module
```

Comment:
We're not using this yet, but the APA102 LED module has a 4th value that can be used to control the overall brightness level by controlling the current flowing to all three LEDs